src/test/lifecycle/common.js

```diff
@@ -569,3 +569,17 @@ exports.test_rule_duplicate_id = async function(Bucket, Key, s3) {
         assert(error.code === 'InvalidArgument', 'Expected InvalidArgument: duplicate ID found in the rules');
     }
 };
+
+exports.test_rule_status_value = async function(Bucket, Key, s3) {
+    const putLifecycleParams = id_lifecycle_configuration(Bucket, Key);
+
+    // set the status value to an invalid value - other than 'Enabled' and 'Disabled'
+    putLifecycleParams.LifecycleConfiguration.Rules[0].Status = 'enabled';
```

Comment:
Since you checked (and fixed it as well) you can add a test with empty status (`""`).

Reply:
I think it is not required, as the test is to check for the values "Enabled" and "Disabled" so it would fail for all the other values(including empty string ""). Also we are handling it with a single condition in `s3_put_bucket_lifecycle.js` so that code is covered with this test.